src/test/java/entities/validity_checkers/WordValidityCheckerRegularTests.java

```diff
@@ -0,0 +1,80 @@
+package entities.validity_checkers;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+
+public class WordValidityCheckerRegularTests {
+
+    /**
+     * We use an implementation of a validity checker where the punctuation value is irrelevant
+     * This allows us to specifically test WordValidityCheckerRegular
+     */
+    public final WordValidityChecker wordValidityChecker =
+            new WordValidityCheckerRegular();
+    public final ValidityCheckerFacade v = new ValidityCheckerFacade(
```

Comment:
Here you should only be testing WordValidityCheckerRegular, with no notion of the facade